### PR TITLE
[ext] Add displaying created date for annotations

### PR DESCRIPTION
**Pull request by Gondragos**

### CODE REVIEW

File: src/common/TimeAgo/TimeAgo.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
+const SECS = 1000;
+const MINS = 60 * SECS;
+
+import { format, formatDistanceToNow } from "date-fns";
```
Comment (hlomzik) [inline src/common/TimeAgo/TimeAgo.js:4]: imports at the most top level

File: src/common/TimeAgo/TimeAgo.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+import { useCallback, useEffect, useMemo, useRef, useState } from "react";
+
+// to_time, [step]
+const STAGES = [
+  [30 * SECS, 30 * SECS],
+  [44 * MINS + 30 * SECS, 1 * MINS],
+  [Number.MAX_SAFE_INTEGER, 30 * MINS],
+];
+
+function * scheduleGen(value = 0) {
```
Comment (hlomzik) [inline src/common/TimeAgo/TimeAgo.js:14]: comments, please. and typescript to understand resulting values

Comment (hlomzik) [reply src/common/TimeAgo/TimeAgo.js:14]: and I think it would be mush simpler to just create a calculation function like `getNextTick(timePassed)`

**Review by hlomzik — CHANGES_REQUESTED** [review]

File: src/common/TimeAgo/TimeAgo.tsx [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+// Each stage is a tuple of limit and step
+// limit is a time from base date, until which stage is active.
+// step is a time-step which we do between two adjusted ticks.
+// stages are chosen to match formatDistanceToNow function from date-fns
+const STAGES: [limit: number, step: number][] = [
+  // from 0 seconds
+  [30 * SECS, 30 * SECS], // to 30 seconds with 30 seconds step
+  [44 * MINS + 30 * SECS, MINS], // to 44 minutes 30 seconds with 1 minute step
+  [Number.MAX_SAFE_INTEGER, 30 * MINS], // to 30 seconds with 30 seconds step
```
Comment (hlomzik) [inline src/common/TimeAgo/TimeAgo.tsx:15]: 30 minutes step till the infinity :)

**Review by hlomzik — APPROVED** [review]

**Review by nicholasrq — APPROVED** [review]

**Pull request merged**